AngularJSLibrary/__init__.py

```diff
@@ -91,6 +92,14 @@ def stripcurly(binding):
 def is_boolean(item):
     return isinstance(item,bool)
 
+def get_driver_obj(lib):
```

Comment:
Would: 
```
@decorator
def driver(self):
    try:
        return self._s2l._current_browser()
    except AttributeError:
        return self._s2l.driver
````
better?
And in the rest of the code just do:
`self.driver` ?

Reply:
Where does @decorator come from? or are you suggesting I create a custom one? Just a little lost with your suggestion but looks cleaner than mine..